pyiron/lammps/control.py

```diff
@@ -641,13 +641,14 @@ def calc_vcsgc(
             append_if_not_present=True
         )
 
-    def measure_mean_value(self, key, every=1, repeat=None, freq=None):
+    def measure_mean_value(self, key, every=1, repeat=None, freq=None, name=None):
         """
             Args:
                 key (str): property to take an average value of (e.g. 'energy_pot' v.i.)
                 every (int): number of steps there should be between two measurements
                 repeat (int): number of measurements for each output (default: n_print/every)
                 freq (int): output frequency (default: n_print)
```

Comment:
I see that this line wasn't code edited in this pull request, but I think we should take the opportunity to remove explicit reference to `n_print`. My concern is that if we do a variable name change in `calc_md` these docs could get out of date. Of course, in principle refactoring the variable name might catch this usage, but since it's in documentation I'm nervous.

Rather, could we simply refer to either the actual code used below (i.e. `self['thermo']`), or --and this would be my preference -- to the logical intent, e.g. `freq (int): Output frequency. (Default is None, which uses the same output frequency as the underlying calculation.)`

Reply:
Further, can we change this from `freq` to `period`?

Reply:
`freq` follows the convention from [this page](https://lammps.sandia.gov/doc/fix_ave_time.html)

Reply:
ok then I'll define a variable inside LAMMPS for `n_print` and will replace `freq` by that.

Reply:
Hmmph. Lammps is wrong. :p 

My preference would be to call the variable something like `period` and make a comment in the docstring that Lammps (*cough*wrongly*cough*) calls it `freq`, but I would also accept leaving it as `freq` and being very explicit in the docstring that it has units of period.

Reply:
Now, I entirely got rid of `Nfreq`, because I don't see why this output should be performed with a different frequency than the output of all the other properties.